src/test/scala/cromwell/CromwellSpec.scala

```diff
@@ -4,12 +4,35 @@ import akka.actor.ActorSystem
 import akka.testkit.{DefaultTimeout, EventFilter, ImplicitSender, TestKit}
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
+import com.typesafe.config.ConfigFactory
+import akka.actor.ActorSystem
+
+object CromwellSpec {
+  val akkaConfigString =
+    """
+      |akka {
+      |  loggers = ["akka.testkit.TestEventListener"]
+      |  event-handlers = ["akka.testkit.TestEventListener"]
+      |  loglevel = "DEBUG"
+      |  actor.debug.receive = on
+      |}
+    """.stripMargin
+
+  def system(name: String) = ActorSystem(name, ConfigFactory.parseString(akkaConfigString))
+}
 
-abstract class CromwellSpec(actorSystem: ActorSystem) extends TestKit(actorSystem) with DefaultTimeout
-with ImplicitSender with WordSpecLike with Matchers with BeforeAndAfterAll with ScalaFutures {
+case class CromwellSpec(actorSystem: ActorSystem)
+  extends TestKit(actorSystem)
+  with DefaultTimeout
+  with ImplicitSender
+  with WordSpecLike
+  with Matchers
+  with BeforeAndAfterAll
+  with ScalaFutures {
 
+  /* TODO: This isn't called anywhere! */
```

Comment:
Good catch!  This should be called in `ThreeStepActorSpec`, I didn't notice its removal in #38.  That should go back in, this is how we're checking that Cromwell correctly determines the calls that have been made runnable.